docs/modules/core/pages/vr/virtualreality.adoc

```diff
@@ -26,6 +28,11 @@ in your settings. To use LWJGL, instead put:
 
 Note that the LWJGL bindings require LWJGL3 (jme3-lwjgl3) to be used.
 
+== Required dependancies
```

Comment:
Dependencies, probably a typo
```suggestion
== Required dependencies
```

Reply:
Thanks, I've corrected this and the other typo you noticed